docsource/maintainer_scripts/setup_new_branch.sh

```diff
@@ -1,54 +1,52 @@
+#!/bin/bash
+set -e
 # Configuration
-export PREV=16.0 OLD=17.0 NEW=18.0
+export PREV=17.0 OLD=18.0 NEW=19.0
 
 git clone https://github.com/OCA/OpenUpgrade --single-branch -b $OLD
 cd OpenUpgrade
+git fetch origin $NEW:$NEW
 
 NODOTPREV=${PREV/\./}         # e.g. 16.0 -> 160
 NODOTDOLD=${OLD/\./}          # e.g. 17.0 -> 170
 NODOTNEW=${NEW/\./}           # e.g. 18.0 -> 180
 ESCAPEDPREV=${PREV/\./\\\.}   # e.g. 16.0 -> 16\.0
 ESCAPEDOLD=${OLD/\./\\\.}     # e.g. 17.0 -> 17\.0
 
-# Create a new empty branch
-git checkout --orphan $NEW
-git rm -rf .
-git commit --allow-empty --no-verify -m "[INIT] $NEW: Initialize branch"
-git push origin $NEW
-
 # Create a dedicated branch
-git checkout -b $NEW-initialize
+git checkout $NEW -b $NEW-initialize
 
-# Recover all the code from the old branch,
-# - Remove coverage docsource file from old version
-# - openupgrade_framework will be migrated in a future PR.
-# - Remove scripts from old version
+# Recover code from the old branch:
+# - bare openupgrade_scripts
+# - openupgrade-specific github actions
 git format-patch --keep-subject --stdout $NEW..$OLD -- \
-    ':!docsource' \
-    ':!openupgrade_framework' \
+    openupgrade_scripts \
     ':!openupgrade_scripts/scripts' \
     ':!openupgrade_scripts/apriori.py' \
+    .github \
+    ':!.github/workflows/pre-commit.yml' \
+    ':!.github/workflows/stale.yml' \
+    ':!.github/workflows/test.yml' \
+    .prettierignore \
 | git am -3 --keep
 
 # Replace No-Dot Syntax (140, 150, ...)
-sed -i "s/$NODOTDOLD/$NODOTNEW/g" .github/workflows/*
-sed -i "s/$NODOTPREV/$NODOTDOLD/g" .github/workflows/*
+sed -i "s/$NODOTDOLD/$NODOTNEW/g" .github/workflows/{documentation-commit.yml,test-migration.yml}
```

Comment:
`test-migration.yml` does not exist in the repository, so it raises an error (or is a problem with my environment)

Reply:
@sergiocorato need any help here? Note this won't work any more now that https://github.com/OCA/OpenUpgrade/pull/5395 is merged, to test this now you'll need to rewind the 19.0 branch on your fork to before this merge, change the remote in the script to use your fork and still add the line from above after line 7